import of role_ownership_grant fails

**Provider Version**

0.75.0, 0.76.0 both have the problem

**Terraform Version**

1.6.3, 1.6.4 both have the problem

**Describe the bug**

When attempting to import a role_ownership_grant, I get an error:

`│ Error: Plugin did not respond
│
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵


Stack trace from the terraform-provider-snowflake_v0.76.0 plugin:

panic: runtime error: index out of range [2] with length 1

goroutine 288 [running]:
github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/resources.ReadRoleOwnershipGrant(0xc000137700, {0x1ced480?, 0xc00076e5b0})
        github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/resources/role_ownership_grant.go:90 +0x797
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0x2054060?, {0x2054060?, 0xc0011e2270?}, 0xd?, {0x1ced480?, 0xc00076e5b0?})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.29.0/helper/schema/resource.go:783 +0x178
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc000bbf7a0, {0x2054060, 0xc0011e2270}, 0xc0014875f0, {0x1ced480, 0xc00076e5b0})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.29.0/helper/schema/resource.go:1089 +0x59e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc00093b1b8, {0x2054060?, 0xc0011e20f0?}, 0xc001307b40)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.29.0/helper/schema/grpc_provider.go:649 +0x4a5
github.com/hashicorp/terraform-plugin-mux/tf5to6server.v5tov6Server.ReadResource({{0x2073560?, 0xc00093b1b8?}}, {0x2054060?, 0xc0011e20f0?}, 0xc0013079c0?)
        github.com/hashicorp/terraform-plugin-mux@v0.12.0/tf5to6server/tf5to6server.go:119 +0x242
github.com/hashicorp/terraform-plugin-mux/tf6muxserver.(*muxServer).ReadResource(0x2053fb8?, {0x2054060?, 0xc000f83da0?}, 0xc0013079c0)
        github.com/hashicorp/terraform-plugin-mux@v0.12.0/tf6muxserver/mux_server_ReadResource.go:35 +0x1b5
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ReadResource(0xc000b16640, {0x2054060?, 0xc000f833e0?}, 0xc00028e900)
        github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov6/tf6server/server.go:787 +0x4b1
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ReadResource_Handler({0x1c998c0?, 0xc000b16640}, {0x2054060, 0xc000f833e0}, 0xc0010b8150, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:431 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002252c0, {0x2063ec0, 0xc0008da680}, 0xc000f9eea0, 0xc000199a40, 0x2d7e8a8, 0x0)
        google.golang.org/grpc@v1.58.2/server.go:1376 +0xdd2
google.golang.org/grpc.(*Server).handleStream(0xc0002252c0, {0x2063ec0, 0xc0008da680}, 0xc000f9eea0, 0x0)
        google.golang.org/grpc@v1.58.2/server.go:1753 +0xa36
google.golang.org/grpc.(*Server).serveStreams.func1.1()
        google.golang.org/grpc@v1.58.2/server.go:998 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.58.2/server.go:996 +0x18c

Error: The terraform-provider-snowflake_v0.76.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.`

**Expected behavior**

Successful import

**Code samples and commands**
(obvious obfuscations have been made)
CLI command:
terraform import 'module.module_name["username"].snowflake_role_ownership_grant.descriptive_name' "role_one_name"
TF resource:
resource "snowflake_role_ownership_grant" "descriptive_name" {
  provider                      = snowflake.security
  depends_on                    = [module.common, snowflake_role.user]
  on_role_name                  = snowflake_role.role_one.name
  to_role_name                  = snowflake_role.role_two.name
  revert_ownership_to_role_name = snowflake_role.role_three.name
  current_grants                = "REVOKE"
}

**Additional context**

Add any other context about the problem here.

--- a/docs/resources/role_ownership_grant.md
+++ b/docs/resources/role_ownership_grant.md
@@ -61,5 +61,5 @@ resource "snowflake_role_ownership_grant" "grant" {
 Import is supported using the following syntax:
 
 ```shell
-terraform import snowflake_role_ownership_grant.example rolename
+terraform import snowflake_role_ownership_grant.example "<on_role_name>|<to_role_name>|<current_grants>"
 ```

--- a/examples/resources/snowflake_role_ownership_grant/import.sh
+++ b/examples/resources/snowflake_role_ownership_grant/import.sh
@@ -1,1 +1,1 @@
-terraform import snowflake_role_ownership_grant.example rolename
+terraform import snowflake_role_ownership_grant.example "<on_role_name>|<to_role_name>|<current_grants>"

